app/src/main/java/org/fossasia/openevent/general/utils/Exceptions.kt

```diff
@@ -0,0 +1,11 @@
+package org.fossasia.openevent.general.utils
+
+/**
```

Comment:
These custom `Exception`s are meant to be used with RxJava's `onError`. We can throw appropriate `Exception`s when we encounter any issues and then check the type using Kotlin's where clause. This provides us with type-safety and makes the code clearer and more explicit about what can go wrong.

Reply:
Great

Reply:
Change name to LocationExceptions and move to location package

Reply:
Ok